packages/react-component-library/src/components/Timeline/TimelineGrid.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import classNames from 'classnames'
+
+import { ComponentWithClass } from '../../common/ComponentWithClass'
+import { extractChildren, timelineChildrenType } from './helpers/children'
+import { StyledHeader } from './partials/StyledHeader'
+import { StyledInner } from './partials/StyledInner'
+import { StyledTimeline } from './partials/StyledTimeline'
+import { TimelineDays } from './TimelineDays'
+import { TimelineHours } from './TimelineHours'
+import { TimelineMonths } from './TimelineMonths'
+import { TimelineRows, TimelineRowsProps } from './TimelineRows'
+import { TimelineSide } from './TimelineSide'
+import { TimelineTodayMarker } from './TimelineTodayMarker'
+import { TimelineToolbar } from './TimelineToolbar'
+import { TimelineWeekColumns } from './TimelineWeekColumns'
+import { TimelineWeeks } from './TimelineWeeks'
+
+export interface ComponentNameProps extends ComponentWithClass {
+  children: timelineChildrenType | timelineChildrenType[]
+  hasSide: boolean
+  hideScaling: boolean
+  hideToolbar: boolean
+}
+
+function getRenderColumns(
+  children: timelineChildrenType | timelineChildrenType[]
+) {
+  const rowsChildren = extractChildren(children, [TimelineRows])
+
+  if (!rowsChildren.length) {
+    return null
```

Comment:
This line is flagged by coverage report is there a test or something that covers this no rows state?